packages/taquito-rpc/src/rpc-client-modules/rpc-cache.ts

```diff
@@ -292,7 +293,8 @@ export class RpcClientCache implements RpcClientInterface {
     const key = this.formatCacheKey(
       this.rpcClient.getRpcUrl(),
       RPCMethodName.GET_NORMALIZED_SCRIPT,
-      [block, address, unparsingMode]
+      [block, address],
+      unparsingMode
```

Comment:
why was this moved out of the array?

Reply:
cuz data is a request body not a param this is the formatCacheKey signature ```private formatCacheKey(
    rpcUrl: string,
    rpcMethodName: RPCMethodName,
    rpcMethodParams: RpcMethodParam[],
    rpcMethodData?: object
  )```